.zuul.yaml

```diff
@@ -2,7 +2,14 @@
     name: kubernetes-sigs/cluster-api-provider-openstack
     check:
       jobs:
-        - cluster-api-provider-openstack-current-acceptance-test-v1.18
+        - cluster-api-provider-openstack-current-acceptance-test-v1.18:
```

Comment:
I would really like to replace OpenLab with Prow. 
As soon as I have [this PR](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/759) green and we have the v1alpha4 and this PR merged there is no need to keep OpenLab:

Reply:
it's different purpose, I want to avoid run CI for non-related test cases..

Reply:
I know, but I will delete this file in my PR because this file is for OpenLab, it doesn't affect our Prow builds.

Reply:
ok, I got it , so I can wait for your PR ready then work on that part

Reply:
Yeah you basically have to do that change in the Prow test-infra repo (where I added the new jobs).
Problem is that Prow does not allow something like this as far as I know. You can say "only run this job when those files change" but you cannot say "don't run this job when only those files change).